utils/frame/try-runtime/remote-externalities/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "remote-externalities"
```

Comment:
should the name be prefixed with try-runtime or something ?

Reply:
This is a standalone project, it is also coming from here: https://github.com/paritytech/substrate-debug-kit/tree/master/remote-externalities

I will move it one level lower on the file tree so that the folder name is not unlogical instead.